index.js

```diff
@@ -31,10 +52,11 @@ module.exports = {
         }
       };
 
-      return new TemplateLinter(tree, {
+      return TemplateLinter.create(tree, {
         annotation: 'TemplateLinter',
         templatercPath: this.project.root + '/.template-lintrc',
-        generateTestFile: this.project.generateTestFile,
+        testGenerator: this._options.testGenerator,
+        group: (this._options.group !== false) ? type : undefined,
```

Comment:
well, they have different meanings though. the CLI addons is only configured once, while the broccoli plugin could potentially run for multiple lint trees. in ESLint that results in `ESLint | app`, `ESLint | tests`, and for some even `ESLint | mirage`, and the same might be true here although from initial testing it seems that `TemplateLint | templates` is the majority of it